lib/hiera/backend.rb

```diff
@@ -8,6 +8,17 @@
 
 class Hiera
   module Backend
+    class LegacyBackendWrapper
+      def initialize(wrapped)
+        @wrapped = wrapped
+      end
+
+      def lookup(key, scope, order_override, resolution_type, recurse_guard)
```

Comment:
Yes. This is the only method implemented by the backends and they don't inherit, include or extend anything.